src/Components/Views/StartMeal/Shared.js

```diff
@@ -19,4 +20,19 @@ export const ScMealType = styled.div`
     &:hover{
     background-color: #950020;
     }
-`;
\ No newline at end of file
+`;*/
+
+export const ScMealType = styled.div`
+    text-align: center;
+    padding: 5%;
+    width: 25%;
+    height 30px;
+    border-style: solid;
+    border-color: 2px;
```

Comment:
2px is not a color

Use 
border:  2px solid #color

Reply:
fixed